android-runtime/runtime/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">runtime</string>
+    <string name="error_call_timeout">Conexão excedeu o tempo de espera.</string>
+    <string name="error_call_failed_without_message">Chamada falhou sem mensagem de erro!</string>
+    <string name="error_unknown">Ocorreu um erro desconhecido na conexão.</string>
+    <string name="error_serialization">Erro de serialização</string>
+    <string name="error_call_code">Erro Fatal (%1$s) - Tente novamente</string>
```

Comment:
To avoid conflict with application strings, we should put a prefix ("sdkgen_") in all string names

Additionally, the app_name string should be removed